src/coreclr/src/jit/compiler.h

```diff
@@ -8979,10 +8979,16 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 //-------------------------- Global Compiler Data ------------------------------------
 
 #ifdef DEBUG
-    static unsigned s_compMethodsCount; // to produce unique label names
-    unsigned        compGenTreeID;
-    unsigned        compStatementID;
-    unsigned        compBasicBlockID;
+private:
+    static LONG s_compMethodID; // to produce unique label names
```

Comment:
Shouldn't this still be `s_compMethodsCount`? After all, it is still a static global count of the number of methods the JIT has compiled. We just happen to often initialize the "method ID" using this.

Reply:
You are right, it is still a static global method count that are JITted but it was (and is) used in producing a unique label names. In fact, I named the instance variable as `compMethodID` and it felt little odd to have name inconsistency during assignment. But I don't have any preference one way or other. I will go ahead and revert the name of this variable.

Reply:
done.